pkg/controller/topology_test.go

```diff
@@ -918,6 +918,7 @@ func TestTopologyAggregation(t *testing.T) {
 			nodeLabels:        []map[string]string{{}, {}, {}},
 			topologyKeys:      nil,
 			expectedRequisite: nil,
+			expectError:       true,
```

Comment:
I probably missed something, but shouldn't all "missing node info" cases expect an error now?

Reply:
If selected node == nil (immediate binding), and no nodes have a CSINode object, then we return no topology keys which falls back to the "no topology specified, provision wherever" case.

I think it is valid to return an error in the case where the returned topology keys are empty.  In order to get to this call, you need the driver to implement the topology capability and have the feature enabled (on by default).